src/pages/design-systems/DesignSystems.stories.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import Page from "./DesignSystems";
+
+export default {
+  title: "Pages/DesignSystems",
+  component: Page,
+  argTypes: {
+    posts: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+};
+
+const fakePosts = [
+  {
+    slug: "/postname",
+    title: "Automatic releases of npm packages with GitHub actions",
+    excerpt:
```

Comment:
In the Gatsby system, excerpt is not a string but a piece of rendered markdown. At the moment it is inserted like a component with certain parameters:

```
<MDXRenderer onlyExcerpt={true} excerptBackup={excerptClean}>
  {post.body}
</MDXRenderer>
```

Preparing a new component which would display excerpt, you need to make it render a piece of HTML/React components. Maybe like

```
excerpt: (<div>
  Hello world
  </div>),
```